src/curated-corpus/pages/SchedulePage/SchedulePage.tsx

```diff
@@ -424,9 +433,13 @@ export const SchedulePage: React.FC = (): JSX.Element => {
       {/** Page Contents Below */}
 
       <Grid container>
-        <Grid item xs={12}>
-          {!data && <HandleApiResponse loading={loading} error={error} />}
+        {!data && (
+          <Grid item xs={12}>
+            <HandleApiResponse loading={loading} error={error} />
+          </Grid>
+        )}
 
+        <Grid item xs={12}>
           {data &&
```

Comment:
Small layout tweak to make sure the closing grid tag that contains the `HandleApiResponse` component is right around it instead of somewhere at the bottom of the page.